rewrite-maven/src/main/java/org/openrewrite/maven/MavenVisitor.java

```diff
@@ -33,14 +33,13 @@
 import static org.openrewrite.internal.StringUtils.matchesGlob;
 
 public class MavenVisitor<P> extends XmlVisitor<P> {
+
     static final XPathMatcher DEPENDENCY_MATCHER = new XPathMatcher("/project/dependencies/dependency");
     static final XPathMatcher PLUGIN_DEPENDENCY_MATCHER = new XPathMatcher("/project/*/plugins/plugin/dependencies/dependency");
```

Comment:
Not strictly related to this PR, but should we possibly also here match plugin dependencies in profiles?
```suggestion
    static final XPathMatcher PLUGIN_DEPENDENCY_MATCHER = new XPathMatcher("//plugins/plugin/dependencies/dependency");
```

Reply:
Note: The corresponding `isPluginDependencyTag()` method is being used by the `UpgradeDependencyVersion` recipe.

Reply:
Thanks! Followed up with https://github.com/openrewrite/rewrite/commit/cfed3dcf0b75b2b56215185c4be7cccee00cd36e just now.